WordPress/Classes/Utility/WPTableViewHandler.h

```diff
@@ -97,4 +96,18 @@
 - (void)refreshCachedRowHeightsForWidth:(CGFloat)width;
 - (void)invalidateCachedRowHeightAtIndexPath:(NSIndexPath *)indexPath;
 
+/**
+ A convenience method for clearing cached row heights and reloading the table view.
+ */
+- (void)refreshTableView;
+
+/**
+ Reloads the table, adjusting the table view's content offset so that the currently
+ visible content stays in place.
+ 
+ The caller should update the tableview's content, (i.e. the fetched results)  
+ in the `tableViewHandlerWillRefreshTableViewPreservingOffset:` delegate method.
+ */
+- (void)refreshTableViewPreservingOffset;
```

Comment:
I'm not sure about testing this particular method, but some of the logic and other methods called could probably be refactored into something testable. I'll have a look.